txpool/txpool.go

```diff
@@ -555,38 +567,60 @@ func (t *TxPool) Underpriced(tx *types.Transaction) bool {
 	return underpriced
 }
 
-func (t *TxPool) Discard(slots uint64, force bool) ([]*types.Transaction, bool) {
+func (t *TxPool) Discard(remaining uint64, force bool) ([]*types.Transaction, bool) {
```

Comment:
The name `remaining` is correct from the context of the below for loop, but not exactly when calling the method. If I were to see this method signature, I wouldn't know what the author meant by `remaining`. I propose `numberOfSlotsToRemove` or something less explicit but still straight to the point.

Reply:
[Renamed](https://github.com/0xPolygon/polygon-sdk/pull/213/commits/0da6d39f81009802cb0ee3253d634dfe9c0204a6)